BUILD

```
@@ -39,3 +40,12 @@ rust_library(
         "@vaticle_dependencies//library/crates:derivative",
     ],
 )
+
+assemble_crate(
+    name = "assemble",
+    target = "typedb_client",
+    description = "Nice crate",
+    homepage = "https://vaticle.com",
```

This is **definitely** something we could accidentally forget to fix later!!

```suggestion
    description = "TypeDB Client API for Rust",
    homepage = "https://github.com/vaticle/typedb-client-rust",
```